src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -400,8 +403,7 @@ def _minimize_corpus_two_step(self, target_path, arguments,
 
     # The dir where merge control file is located must persist for both merge
     # steps. The second step re-uses the MCF produced during the first step.
-    merge_control_file_dir = self._create_temp_corpus_dir('mcf_tmp_dir')
-    self._merge_control_file = os.path.join(merge_control_file_dir, 'MCF')
+    self._merge_control_file = os.path.join(fuzzer_utils.get_temp_dir(), 'MCF')
```

Comment:
looks like `fuzzer_utils.get_temp_dir()` isn't persistent across two runs (see the comment above), let me try reverting this part